ansible/roles/system_config/templates/passenger.j2

```diff
@@ -0,0 +1,4 @@
+### Begin automatically installed Phusion Passenger config snippet ###
+passenger_root /usr/lib/ruby/vendor_ruby/phusion_passenger/locations.ini;
+passenger_ruby /home/decidim/.rbenv/versions/2.6.6/bin/ruby;
```

Comment:
Is it possible to use a dynamically set ruby version here?

Reply:
It is not necessary to define a ruby version dynamically in this context because we are going to deploy version 0.23.6 of Decidim which uses version 2.6.6 of Ruby

Reply:
Yes but we should read it from .ruby-version

Reply:
Thanks